ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/AbstractPendingTransactionsSorter.java

```diff
@@ -248,8 +264,7 @@ public void selectTransactions(final TransactionSelector selector) {
           switch (result) {
             case DELETE_TRANSACTION_AND_CONTINUE:
               transactionsToRemove.add(transactionToProcess);
-              signalInvalidAndGetDependentTransactions(transactionToProcess)
-                  .forEach(transactionsToRemove::add);
+              signalInvalidTransaction(transactionToProcess);
```

Comment:
Renamed as suggested.
Do not really like this method, was a way to improve the legacy txpool, luckily it will go away when the layered txpool will be the default